tests/utils/axis-utils-tests.js

```diff
@@ -32,7 +32,7 @@ test('axis-utils #getTicksTotalFromSize', t => {
 
 test('axis-utils #getTickValues', t => {
   const scale = scaleLinear().domain([0, 1]).range(['red', 'blue']);
-  t.deepEqual(getTickValues(scale, 10, false),
+  t.deepEqual(getTickValues(scale, 10, false).map(d => Math.round(d * 1000) / 1000),
```

Comment:
Looks like this was fixed in d3-array 1.2.0 . Perhaps we need to update our dependencies to 1.2 , and update d3-scales to use d3-array 1.2 .